### PR TITLE
Added a few more distros, bsd, reorg, etc.

### PR DESCRIPTION
**Pull request by Menelkir**
Sorry for the shortly second request, but I think it's a little bit improved now.
1) I've added two more distributions, changed the devuan icon and added a new line to be a little more organized
2) Changed the slackware link to https
3) Added project-trident to bsd

### CODE REVIEW

**Comment by muellermartin** [comment]
Thanks for contributing even more :) I mostly don't know these distributions and I don't know whether it is a good strategy to include all distributions without systemd rather than focusing on popular and therefore well maintained distros. On the other hand this website might help promoting them, so it's a good idea to have them listed up to a point where this list gets too large.

The URL change to the Slackware website is an important addition, but the website sadly redirects from HTTPS to HTTP. Nevertheless this might change, so it's probably good to prefer the HTTPS link. And we can check the link with the `checklinks.py` whether it still works.

The only thin I don't like is the line break, because it highly depends on the screen resolution. On mobile devices this will certainly look weird. Therefore I will probably not add it and think of a better way to organize the overview of the alternative distros.

**Comment by Menelkir** [comment]
The distributions I've tried to see distributions that are more independent instead of based in other ones, otherwise the list will be huge since now there's a couple of devuan-based distributions and also a lot of slackware-based ones.

About the line break, that's a good point, I didn't checked the mobile version to see if gets ugly.

File: htdocs/index.html [hunk 1]
```diff
@@ -73,9 +73,12 @@ <h2>How do I get rid of systemd?</h2>
 			<li><a href="https://artixlinux.org/"><img src="img/artix.png" alt="Artix" /></a></li>
 			<li><a href="https://www.gnu.org/software/guix/"><img src="img/guix.png" alt="Guix" /></a></li>
 			<li><a href="https://gentoo.org/"><img src="img/gentoo.png" alt="Gentoo" /></a></li>
+			<br>
```
Comment (muellermartin) [inline htdocs/index.html]: I suggest that the line break gets removed as it likely will break weirdly on mobile devices.

Comment (Menelkir) [reply htdocs/index.html]: I've forgot that. Done.

**Comment by Menelkir** [comment]
I've removed the line break on my master, not sure why isn't reflecting here.

**Pull request merged**